Update cross region instructions for private link (#121748)
Azure/azure-content: articles/azure-monitor/containers/kubernetes-monitoring-enable.md

## Enable Prometheus and Grafana
Use one of the following methods to enable scraping of Prometheus metrics from your cluster and enable Managed Grafana to visualize the metrics. See [Link a Grafana workspace](../../managed-grafana/quickstart-managed-grafana-portal.md) for options to connect your Azure Monitor workspace and Azure Managed Grafana workspace.

> > [!NOTE] If you have a single Azure Monitor Resource that is private-linked, then Prometheus enablement will not work if the AKS cluster and Azure Monitor Workspace are in different regions.

Reviewer: ```suggestion
> [!NOTE] 
> If you have a single Azure Monitor Resource that is private-linked, then Prometheus enablement will not work if the AKS cluster and Azure Monitor Workspace are in different regions.
```

> > [!NOTE] If you have a single Azure Monitor Resource that is private-linked, then Prometheus enablement will not work if the AKS cluster and Azure Monitor Workspace are in different regions.

Reviewer 2: Thanks @Sohamdg081992. Please add this link in the doc too for the full instructions for private link. https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/private-link-data-ingestion
Also please remove the unintended white space changes

Author: Thanks @rashmichandrashekar! I have made these changes. My VSC was trimming additional whitespaces which was causing this problem, disabled it now. :)

## Enable Prometheus and Grafana
Use one of the following methods to enable scraping of Prometheus metrics from your cluster and enable Managed Grafana to visualize the metrics. See [Link a Grafana workspace](../../managed-grafana/quickstart-managed-grafana-portal.md) for options to connect your Azure Monitor workspace and Azure Managed Grafana workspace.

> [!NOTE] 

> > If you have a single Azure Monitor Resource that is private-linked, then Prometheus enablement will not work if the AKS cluster and Azure Monitor Workspace are in different regions.
> > The configuration needed for Prometheus add-on is not available cross region because of the private link constraint.
> > To resolve this, just create a new DCE in the AKS cluster location and a new DCRA (association) in same AKS cluster region. Associate new dce with aks cluster and name the new association(DCRA) as configurationAccessEndpoint.
> > For full instructions on how to configure the DCEs associated with your Azure Monitor workspace to use a Private Link for data ingestion, please refer this [doc](../essentials/private-link-data-ingestion.md).

Reviewer: ```suggestion
> If you have a single Azure Monitor Resource that is private-linked, then Prometheus enablement won't work if the AKS cluster and Azure Monitor Workspace are in different regions.
> The configuration needed for the Prometheus add-on isn't available cross region because of the private link constraint.
> To resolve this, create a new DCE in the AKS cluster location and a new DCRA (association) in the same AKS cluster region. Associate the new DCE with the AKS cluster and name the new association (DCRA) as configurationAccessEndpoint.
> For full instructions on how to configure the DCEs associated with your Azure Monitor workspace to use a Private Link for data ingestion, see [Use a private link for Managed Prometheus data ingestion](../essentials/private-link-data-ingestion.md).
```
